common/src/chain/transaction/output/output_value.rs

```diff
@@ -16,27 +16,30 @@
 use serialization::{Decode, Encode};
 
 use crate::{
-    chain::tokens::{NftIssuance, TokenData, TokenIssuance, TokenTransfer},
+    chain::tokens::{NftIssuance, TokenData, TokenId, TokenIssuance, TokenTransfer},
     primitives::Amount,
 };
 
 #[derive(Debug, Clone, PartialEq, Eq, Encode, Decode, serde::Serialize)]
 pub enum OutputValue {
     Coin(Amount),
+    // FIXME: add check that new tokens cannot be issued after upgrade;
+    // transfers of existing v0 tokens are allowed?
     Token(Box<TokenData>),
```

Comment:
You can safely rename this arm to TokenV0

Reply:
But again, this will just make the diff bigger... it should be done at some point. Perhaps in a separate PR.

Reply:
Renamed at once, the change is not that big but make the code much more clear